app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsViewModel.kt

```diff
@@ -32,9 +32,9 @@ class AdministratorControlsViewModel @Inject constructor(
   private val fragment: Fragment,
   private val oppiaLogger: OppiaLogger,
   private val profileManagementController: ProfileManagementController,
-  @EnableEditAccountsOptionsUi
-  private val enableEditAccountsOptionsUi: PlatformParameterValue<Boolean>,
-  @LearnerStudyAnalytics private val learnerStudyAnalytics: PlatformParameterValue<Boolean>
+  @LearnerStudyAnalytics private val learnerStudyAnalytics: PlatformParameterValue<Boolean>,
+  @AutomaticallyUpdateTopic private val automaticallyUpdateTopic: PlatformParameterValue<Boolean>
```

Comment:
Can you also add tests for verifying that the option correctly shows/hides based on the parameter value?

Reply:
Hi @BenHenning , how do I go about this? Thanks

Reply:
@Ryggs take a look at tests that use https://github.com/oppia/oppia-android/blob/2ff2945934a1321cb931c5256b3a8bb5fcefd63f/testing/src/main/java/org/oppia/android/testing/platformparameter/TestPlatformParameterModule.kt#L178. There are tests to verify the parameter-enabled and parameter-disabled states.

Reply:
Right, I can pick up from here

Reply:
Addressed